javascript/modules/motion/index.js

```diff
@@ -0,0 +1,37 @@
+import { createClient } from '@unabridged/motion';
+import { createConsumer } from "@rails/actioncable"
+
+export default createClient({
+
+  // To avoid creating a second websocket, make sure to reuse the application's
+  // ActionCable consumer. If you are not otherwise using ActionCable, you can
+  // remove this line and the corresponding import.
+  consumer: createConsumer(),
```

Comment:
FWIW, this is what Motion does if you choose to omit `consumer` here entirely. :smile: